examples/langevin/montecarlo_lj.yml

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env yason.py
+energy:
+    - nonbonded_coulomblj:
+        coulomb: {epsr: 80, type: plain}
```

Comment:
skip coulomb since the particle charge is zero?

Reply:
Done in both langevin and montecarlo examples.